azure-pipelines.yml

```diff
@@ -0,0 +1,61 @@
+trigger:
+- V2
+
+pr:
+  branches:
+    include:
+      - V2
+
+jobs:
+- job: 'Test'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python37:
+        python.version: '3.7'
+#      Python38:
```

Comment:
should be also run on version 3.6?

Reply:
Possibly. Let's see how long it takes first and decide if we want to add more options?

Reply:
agree